README.md

```diff
@@ -35,6 +35,7 @@ Creating this web application will provide exposure to:
 Model -
```

Comment:
citing line ~20: ```[ ] All code submissions are peer reviewed via GitHub PR by at least two members of the team, and master is always in a stable state (tests passed, site functions)``` How many team members are on your team? Re-phrase this so that it reflects the actual structure of your team. Maybe a more appropriate phrasing is ```[ ] All code submissions are peer reviewed via Prrr before merging in to Master and Master is always in a stable state (tests passed, site functions)``` Also I don't see any specs about tests. Are you going to be writing any? If not remove them from being referenced in your specs (though I would recommend striving for writing tests always.